PurchasesHybridCommonUI.podspec

```diff
@@ -0,0 +1,35 @@
+Pod::Spec.new do |s|
+  s.name             = "PurchasesHybridCommonUI"
+  s.version          = "8.11.2"
+  s.summary          = "Common files for hybrid SDKs for RevenueCat UI"
+
+  s.description      = <<-DESC
+                       Save yourself the hastle of implementing a subscriptions backend. Use RevenueCat instead https://www.revenuecat.com/
+                       DESC
+
+  s.homepage         = "https://www.revenuecat.com/"
+  s.license          =  { :type => 'MIT' }
+  s.author           = { "RevenueCat, Inc." => "support@revenuecat.com" }
+  s.source           = { :git => "https://github.com/revenuecat/purchases-hybrid-common.git", :tag => s.version.to_s }
+  s.documentation_url = "https://docs.revenuecat.com/"
+
+  s.framework      = 'StoreKit'
+  s.framework      = 'SwiftUI'
+
+  s.dependency 'RevenueCat', '4.32.0'
+  s.dependency 'RevenueCatUI', '4.32.0'
```

Comment:
Does this need both defined? Or will the `RevenueCat` get to the project transitively